tests/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# NOTE: We're not using CTest here as just adds an extra layer around QuickTest
```

Comment:
At least for now all the QML tests are run via one executable, which already displays pass/fail progress, and can be run via `make check`. Using CTest would hide the individual QuickTest results by default and doesn't really provide much advantage now.